test/micro/org/openjdk/bench/java/text/DateFormatterBench.java

```diff
@@ -51,27 +53,57 @@
 public class DateFormatterBench {
 
     private Date date;
-
     private Object objDate;
+    private String dateStr;
+    private String timeStr;
+
+    private final String datePattern = "EEEE, MMMM d, y";
+    private final String timePattern = "h:mm:ss a zzzz";
```

Comment:
Nit: These can be static fields (then the field names need to be all capital letters)